### ISSUE TITLE
invalid_client with service account

### ISSUE THREAD

**Issue by va-b** [googleads/google-ads-dotnet#211]
Google.Ads.GoogleAds v4.0.0
Net Core 3.1
win10-x64

**Actions taken**
`            var client = new GoogleAdsClient(new GoogleAdsConfig
            {
                DeveloperToken = DeveloperToken,
                OAuth2Mode = OAuth2Flow.SERVICE_ACCOUNT,
                OAuth2SecretsJsonPath = Path.Combine(Directory.GetCurrentDirectory(), "responsive-hall-284114-1d4e3cda539a.json"),
                OAuth2PrnEmail = AdWordsAccountEmail
            });`
**Expected result**
Works properly as Google.AdWords with identical config

**Actual result**
Error:"invalid_client", Description:"The OAuth client was not found.", Uri:""

**Comment by keeblebogdan** [comment]
I have the same issue

**Comment by morsanu** [comment]
Same here.

**Comment by va-b** [comment]
@keeblebogdan   No. Finally we have agreed with the manager to use regular accounts

**Comment by AnashOommen** [comment]
That's strange, I am pretty sure this got fixed in 4.0.0. Could you send me a way to replicate this issue by mailing a json file and account email to googleadsapi-support@google.com? Please address the email to Attn:Anash so it reaches me.

**Comment by AndrewMBurke** [comment]
The issue has been identified and is fixed in the upcoming V5 release.

**Comment by keeblebogdan** [comment]
> The issue has been identified and is fixed in the upcoming V5 release.

Thank you!

**Comment by AnashOommen** [comment]
This is now fixed. Please reopen if you see it recurring.
